xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1022,7 +1022,7 @@ void CPVRClients::StartChannelScan(void)
   long perfCnt = XbmcThreads::SystemClockMillis();
 
   /* stop the supervisor thread */
-  g_PVRManager.StopUpdateThreads();
+  g_PVRManager.Stop();
```

Comment:
"no longer needed" as in it shouldn't crash because of it now. other logic may still need to be changed to handle this situation.